woocommerce/woocommerce-android
# Store creation flow exit improvements

This PR adds some UX improvements for exiting the store creation flow:

1. Tapping on the `Back` link on the first page of the survey exits the flow
2. Tapping on the back button on the toolbar shows a confirmation dialog

<image src="https://user-images.githubusercontent.com/1522856/199287448-97e3e905-4f97-45d3-8363-06fb4cf7181b.png" width="300" />

**To test:**
1. Start the store creation flow
2. Tap on the `Back` link button on the page
3. Notice the store creation flow is closed
4. Start the store creation flow again
5. Tap on the toolbar back navigation button
6. Notice a confirmation dialog is show
7. Tap OK and notice the creation flow is closed
